backend/config/config.js

```diff
@@ -1,20 +1,20 @@
 module.exports = {
   development: {
-    dialect: "sqlite",
-    storage: "./src/db/databases/dev.sqlite"
+    dialect: 'sqlite',
+    storage: './src/db/databases/dev.sqlite',
   },
   qa: {
-    host: "drona.db.elephantsql.com",
-    database: "artamutj",
-    username: "artamutj",
-    password: "r3AAc-798qno4Ba8FT_9DtLqHJyLLQKh",
-    dialect: "postgres"
+    host: 'drona.db.elephantsql.com',
+    database: 'artamutj',
+    username: 'artamutj',
+    password: 'r3AAc-798qno4Ba8FT_9DtLqHJyLLQKh',
+    dialect: 'postgres',
   },
   prod: {
     host: process.env.DATABASE_URL,
     database: process.env.DATABASE_NAME,
     username: process.env.DATABASE_USERNAME,
     password: process.env.DATABASE_PASSWORD,
-    dialect: "postgres"
-  }
+    dialect: 'postgres',
+  },
```

Comment:
Eu havia incluído algums configurações para forçar o uso de UTC ontem. Contudo elas não estão mais aqui